deps/std/bytes.ts

```diff
@@ -1 +1,15 @@
 export * from "https://deno.land/std@0.185.0/bytes/mod.ts"
+
```

Comment:
alternatively we can still have a `bytes.ts` file under util and have that file re-export the std bytes library. might be cleaner than _polluting_ the std namespace?